broker/fsm.go

```diff
@@ -1,9 +1,12 @@
 package broker
 
 import (
+	"encoding/json"
 	"io"
 
 	"github.com/hashicorp/raft"
```

Comment:
it'd be nice if we could make it so the broker pkg didn't depend on raft (and same for serf). we'd have to do some wrapping for things like *raft.Log for the apply method though.